debug-plugin.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
```

Comment:
Suggest `set -euf -o pipefail` here to make the bash script safer.

Reply:
great idea, done